Compilation failure using gcc 10.2

```
rpcs3/Emu/Cell/Modules/cellSpurs.cpp:1061:98: warning: comparison between ‘enum CellError’ and ‘enum CellNotAnError’ [-Wenum-compare]
 1061 |  if (process_is_spu_lock_line_reservation_address(spurs.addr(), SYS_MEMORY_ACCESS_RIGHT_SPU_THR) != CELL_OK)
      |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~
rpcs3/Emu/Cell/Modules/cellSpurs.cpp:1061:98: error: comparison between types ‘CellError’ and ‘CellNotAnError’ [-Werror=sign-compare]
cc1plus: some warnings being treated as errors
```
Caused by https://github.com/RPCS3/rpcs3/pull/9030
CC @elad335

--- a/rpcs3/Emu/Cell/Modules/cellSpurs.cpp
+++ b/rpcs3/Emu/Cell/Modules/cellSpurs.cpp
@@ -1058,7 +1058,7 @@ s32 _spurs::initialize(ppu_thread& ppu, vm::ptr<CellSpurs> spurs, u32 revision, 
 		return CELL_SPURS_CORE_ERROR_INVAL;
 	}
 
-	if (process_is_spu_lock_line_reservation_address(spurs.addr(), SYS_MEMORY_ACCESS_RIGHT_SPU_THR) != CELL_OK)
+	if (process_is_spu_lock_line_reservation_address(spurs.addr(), SYS_MEMORY_ACCESS_RIGHT_SPU_THR))
 	{
 		return CELL_SPURS_CORE_ERROR_PERM;
 	}

